### ISSUE TITLE
Dependencies for plans that use libraries not in JesterJ

### ISSUE THREAD

**Issue by nsoft** [nsoft/jesterj#146]
Current case is a plan that involves a processor that pools resources. Commons Pool is not one of our dependencies, so 

```
Caused by: java.lang.ClassNotFoundException: org.apache.commons.pool2.PooledObjectFactory
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
```

One could complexify the command line with classpath declarations, but that's fiddly and error prone. Perhaps we can accept UnoJar packed uber jars.

**Comment by nsoft** [comment]
Also of course the jar can be pushed into the JesterJ jar but that's also ugly. (and currently required for JDBC use cases).

**Comment by nsoft** [comment]
Fixed on JDK 11 branch but see #116 for requirements to get that going.
